projects/ffmpeg/build.sh

```diff
@@ -66,13 +71,24 @@ make -j$(nproc) all
 make install
 
 cd $SRC/libvpx
-LDFLAGS="$CXXFLAGS" ./configure --prefix="$FFMPEG_DEPS_PATH" \
-    --disable-examples --disable-unit-tests \
-    --size-limit=12288x12288 \
-    --extra-cflags="-DVPX_MAX_ALLOCABLE_MEMORY=1073741824"
-make clean
-make -j$(nproc) all
-make install
+if [[ "$ARCHITECTURE" == i386 ]]; then
```

Comment:
No need to duplicate the code, you can simply add `--target=x86-linux-gcc` in the condition.